src/style.css

```diff
@@ -2,0 +1,90 @@
+body {
+  font-family: ' Avenir', Arial, sans-serif;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  font-family: ' Avenir', sans-serif;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+.search-box {
+  position: relative;
+  margin: 20px 0;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+}
+
+.search-box--big {
+  height: 70px;
+}
+
+.search-box--small {
+  height: 42px;
+}
+
+.search-box--big::before {
+  position: absolute;
+  content: '';
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search-box--small::before {
+  position: absolute;
+  content: '';
+  background-image: url('./images/search-small.svg');
+  background-repeat: no-repeat;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search-txt {
+  border-radius: 4px;
```

Comment:
The '.search-txt' class is applying a border-radius, which is not needed here since this class is used for text elements and not for any box/container elements.